eclipse-scout-core/src/scrollbar/scrollbars.js

```diff
@@ -127,6 +140,190 @@ export function _installNativeInternal($container, options) {
   $container.css('-webkit-overflow-scrolling', 'touch');
 }
 
+function installScrollShadow($container, session, options) {
+  let scrollShadowStyle = _computeScrollShadowStyle(options);
+  if (scrollShadowStyle.length === 0) {
+    return;
+  }
+  let $shadow = $container.afterDiv('scroll-shadow');
+  $shadow.toggleClass('large', scrollShadowStyle.indexOf('large') > -1);
+  $shadow.toggleClass('gradient', scrollShadowStyle.indexOf('gradient') > -1);
+  $shadow.data('scroll-shadow-parent', $container);
+  $container.data('scroll-shadow', $shadow);
+  $container.data('scroll-shadow-style', scrollShadowStyle);
+  $container.on('scroll', () => updateScrollShadow($container));
+  updateScrollShadow($container);
+  _installMutationObserver(session);
+  _installIntersectionObserver();
+  intersectionObserver.observe($container[0]);
+}
+
+function uninstallScrollShadow($container, session) {
+  let $shadow = $container.data('scroll-shadow');
+  if ($shadow) {
+    $shadow.remove();
+  }
+  intersectionObserver.unobserve($container[0]);
```

Comment:
remove 'scroll-shadow' and 'scroll-shadow-style' from $container data?
And remove 'scroll' event listener from container?